docs/reference/query-dsl/parent-id-query.asciidoc

```diff
@@ -1,68 +1,113 @@
 [[query-dsl-parent-id-query]]
 === Parent Id Query
 
-The `parent_id` query can be used to find child documents which belong to a particular parent.
-Given the following mapping definition:
+Returns child documents <<parent-join,joined>> to a specific parent document.
+You can use a <<parent-join,join>> field mapping to create parent-child
+relationships between documents in the same index.
 
+[[parent-id-query-ex-request]]
+==== Example request
+
+[[parent-id-index-setup]]
+===== Index setup
+To use the `parent_id` query, your index must include a <<parent-join,join>>
+field mapping. To see how you can set up an index for the `parent_id` query, try
+the following example.
+
+. Create an index with a <<parent-join,join>> field mapping.
++
+--
 [source,js]
---------------------------------------------
-PUT my_index
+----
+PUT /my-index
 {
-  "mappings": {
-    "properties": {
-      "my_join_field": {
-        "type": "join",
-        "relations": {
-          "my_parent": "my_child"
+    "mappings": {
+        "properties" : {
+            "my-join-field" : {
+                "type" : "join",
+                "relations": {
+                    "my-parent": "my-child"
+                }
+            }
         }
-      }
     }
-  }
 }
 
-PUT my_index/_doc/1?refresh
+----
+// CONSOLE
+// TESTSETUP
+--
+
+. Index a parent document with an ID of `1`.
++
+--
+[source,js]
+----
+PUT my-index/_doc/1?refresh
 {
-  "text": "This is a parent document",
-  "my_join_field": "my_parent"
+  "text": "This is a parent document.",
+  "my-join-field": "my-parent"
 }
+----
+// CONSOLE
+--
 
-PUT my_index/_doc/2?routing=1&refresh
+. Index a child document of the parent document.
++
+--
+[source,js]
+----
+PUT my-index/_doc/2?routing=1&refresh
 {
-  "text": "This is a child document",
+  "text": "This is a child document.",
   "my_join_field": {
-    "name": "my_child",
+    "name": "my-child",
     "parent": "1"
   }
 }
-
---------------------------------------------
+----
 // CONSOLE
-// TESTSETUP
+--
+
+[[parent-id-query-ex-query]]
+===== Example query
+
+The following search returns child documents for a parent document with an ID of
+`1`.
 
 [source,js]
---------------------------------------------------
-GET /my_index/_search
+----
+GET /my-index/_search
```

Comment:
should we have a unified format in all examples:  either put `/` before index name or not? `/my-index/_search` -> `my-index/_search`

Reply:
Thank you for catching this. I added leading slashes with 647110c.